scripts/download-realm.js

```diff
@@ -72,7 +72,23 @@ function printProgress(input, totalBytes, archive) {
     });
 }
 
+function copyLocal(archive, destination) {
+    const archivePath = `${process.env.HOME}/${archive}`;
+    const exists = fs.existsSync(archivePath);
+    if (exists) {
+        console.log(`Skipping download for ${archive}. Local archive available at ${archivePath}`);
+        fs.copySync(archivePath, destination);
+        return true;
+    }
+
+    return false;
+}
+
 function download(serverFolder, archive, destination) {
+    if (copyLocal(archive, destination)) {
+        return;
```

Comment:
This should return a promise (Just a reminder, same as https://github.com/realm/realm-js/pull/3096).